package.json

```diff
@@ -10,6 +10,7 @@
     "@fortawesome/react-fontawesome": "^0.1.4",
     "bulma": "^0.9.0",
     "creditkey-js": "file:../creditkey-js",
+    "currency.js": "^2.0.3",
```

Comment:
@widox would this lib be worthwhile on the checkout ui?

Reply:
👍 prolly